x-pack/plugins/ml/server/routes/saved_objects.ts

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { wrapError } from '../client/error_wrapper';
+import { RouteInitialization } from '../types';
+import { checksFactory } from '../saved_objects';
+import { jobsAndSpaces, repairJobObjects } from './schemas/saved_objects';
+
+/**
+ * Routes for job saved object management
+ */
+export function savedObjectsRoutes({ router, mlLicense }: RouteInitialization) {
```

Comment:
These new routes should be added to `ml/server/routes/apidoc.json` to add them to the API docs.

Reply:
6c5756fd5c023f44cc0e36b48f5986dd94dba23f